### ISSUE TITLE
[feat]: Add ComposedChart

### ISSUE THREAD

**Issue by javicrg** [shadcn-ui/ui#4187]
### Feature description

Charts have just been added, and Recharts' library actually has a ComposedChart component to be able to merge charts on the same grid, it would be nice to implement it.

### Additional Context

Additional details here...

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues and PRs

**Comment by shadcn** [comment]
I'm working on this. It's on the roadmap.

**Comment by jonnystalnaker** [comment]
Adding a dot here to see when this is completed. Would love to switch over completely, but this is required for one of my clients dashboards.

**Comment by Gobot1234** [comment]
Was there a reason this was closed. It still doesn't seem to work

**Comment by omersanli17** [comment]
Is there any updates to built native shadcn component instead of LineBarAreaComposedChart of Recharts ?

**Comment by omersanli17** [comment]
@shadcn  @alekfestekjian Exactly

**Comment by JoaquinPalacios** [comment]
It shouldn't have been closed. It still not working yet.

**Comment by omersanli17** [comment]
Yeah, it should be added native multi-axis shadcn charts component.

**Comment by sina-ss** [comment]
Still no update?

**Comment by raulmouzo** [comment]
Need this!

**Comment by derekbking** [comment]
Would love to see this as well!
